homeassistant/components/laundrify/sensor.py

```diff
@@ -0,0 +1,108 @@
+"""Platform for sensor integration."""
+
+import logging
+
+from laundrify_aio import LaundrifyDevice
+from laundrify_aio.exceptions import LaundrifyDeviceException
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfEnergy, UnitOfPower
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import LaundrifyUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, config: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Add power sensor for passed config_entry in HA."""
+
+    coordinator: LaundrifyUpdateCoordinator = hass.data[DOMAIN][config.entry_id][
+        "coordinator"
+    ]
+
+    sensor_entities: list[LaundrifyPowerSensor | LaundrifyEnergySensor] = []
+    for device in coordinator.data.values():
+        sensor_entities.append(LaundrifyPowerSensor(device))
+        sensor_entities.append(LaundrifyEnergySensor(coordinator, device))
+    if sensor_entities:
+        async_add_entities(sensor_entities)
```

Comment:
```suggestion
    async_add_entities(sensor_entities)
```

Reply:
Addressed in https://github.com/home-assistant/core/pull/121378/commits/d0fd601172b3cfd74f56e426135ff6a61eff42f3